app/views/home/_funding_levels.html.erb

```diff
@@ -0,0 +1,21 @@
+<section>
+  <h3>Fundraising Progress</h3>
+
+  <%= render "funding_progress_bar" %>
+
```

Comment:
I've added this, and put it before the infographic.  What do you think of its placement now that you can [see it on the page](https://rubytogether-staging-pr-29.herokuapp.com)?